packages/block-editor/src/components/writing-flow/use-multi-selection.js

```diff
@@ -122,6 +122,12 @@ export default function useMultiSelection() {
 				return;
 			}
 
+			// The block refs might not be immediately available
+			// when dragging blocks into another block.
+			if ( ! startRef.current || ! endRef.current ) {
+				return;
```

Comment:
It sounds like we'll need a state based alternative to `useBlockRef`. Updating the refs won't trigger a re-render here, so nothing will happen.

It's good enough for now though, if it prevents an error.

Reply:
I tried using `useBlockElement`, which I think is the state-based alternative but was getting the same results.